### PR TITLE
readme adjustment

### PR DESCRIPTION
**Pull request by aklawonn**
The readme.md file was adjusted to clarify which solvers/inference methods the package provides.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,10 +6,12 @@
 [![coverage](https://img.shields.io/endpoint?url=https://gist.githubusercontent.com/aklawonn/5eb707145cc7d75de25b43d25b13c972/raw/probeye_main_coverage.json)](https://en.wikipedia.org/wiki/Code_coverage)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-This package provides a transparent and easy-to-use framework for solving parameter estimation problems (i.e., [inverse problems](https://en.wikipedia.org/wiki/Inverse_problem)) in a characteristic two-step approach. 
+This package provides a transparent and easy-to-use framework for solving parameter estimation problems (i.e., [inverse problems](https://en.wikipedia.org/wiki/Inverse_problem)) chiefly via [sampling](https://ermongroup.github.io/cs228-notes/inference/sampling/) methods in a characteristic two-step approach.
```
Comment (joergfunger) [inline README.md:9]: chiefly -> primarily?

File: docs/index.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
 # probeye
 
-This package provides a transparent and easy-to-use framework for solving parameter estimation problems (i.e., [inverse problems](https://en.wikipedia.org/wiki/Inverse_problem)) in a characteristic two-step approach. 
+This package provides a transparent and easy-to-use framework for solving parameter estimation problems (i.e., [inverse problems](https://en.wikipedia.org/wiki/Inverse_problem)) chiefly via [sampling](https://ermongroup.github.io/cs228-notes/inference/sampling/) methods in a characteristic two-step approach.
```
Comment (joergfunger) [inline docs/index.md:3]: chiefly -> primarily

**Review by joergfunger — COMMENTED** [review]
just the wording suggestion.

**Review by joergfunger — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (aklawonn) [reply README.md:9]: Sure, I'll change that.

File: docs/index.md (hunk 2 above)
Comment (aklawonn) [reply docs/index.md:3]: Sure, I'll change that.

**Pull request merged**